### PR TITLE
chore:update config files

### PR DESCRIPTION
**Pull request by SarahHouben**

### CODE REVIEW

File: base.js [hunk 1]
```diff
@@ -6,6 +6,7 @@ module.exports = {
     "eslint:recommended",
     "plugin:@typescript-eslint/eslint-recommended",
     "plugin:@typescript-eslint/recommended",
+    "plugin: import/order",
```
Comment (pradel) [inline base.js]: @SarahHouben there is a space there, looks wrong no?

**Review by pradel — COMMENTED** [review]
With this pr, I think we can remove `base.production.js` since we will never use it, what do you think?

**Comment by SarahHouben** [comment]
> With this pr, I think we can remove `base.production.js` since we will never use it, what do you think?

Yes, you're right. I'll remove the file.

File: base.js (hunk 1 above)
Comment (SarahHouben) [reply base.js]: True! Removed the space.

File: package.json [hunk 2]
```diff
@@ -7,7 +7,9 @@
     "base.js",
     "base.production.js",
```
Comment (pradel) [inline package.json]: let's delete this line?

Comment (SarahHouben) [reply package.json]: Yes, of course.

**Pull request merged**